tests/page_test.go

```diff
@@ -1699,3 +1699,59 @@ func TestShadowDOMAndDocumentFragment(t *testing.T) {
 		})
 	}
 }
+
+func TestPageTargetBlank(t *testing.T) {
```

Comment:
Thanks for adding a test 🙇 Should we also add a link as a comment to the "new tab never loads" issue?

Reply:
I don't think we need to do that. We haven't done that for any of our other fixes. People can browser the git history if they want to find that information.

What's your reasoning?

Reply:
Alright. No worries.